SCons/Tool/gfortran.xml

```diff
@@ -26,23 +26,10 @@ See its __doc__ string for a discussion of the format.
 <tool name="gfortran">
 <summary>
 <para>
-Sets construction variables for the GNU F95/F2003 GNU compiler.
+Sets construction variables for the GNU Fortran compiler.
+Calls the &t-link-fortran; Tool module to set variables.
 </para>
 </summary>
-<sets>
```

Comment:
Doesn't set these variables?

Reply:
See added comment lines 29-30 - yes, they get set, but this intends to refer the settings to the place they happen.